### PR TITLE
[INLONG-7542][Dashboard] File source supports selecting clusters

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes https://github.com/apache/inlong/issues/7542

### Verifying this change

image

### CODE REVIEW

**Review by leezng — CHANGES_REQUESTED** [review]
Don't use temporary variables

File: inlong-dashboard/src/locales/en.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   "meta.Sources.Type": "Type",
   "meta.Sources.File.SerializationType": "File type",
   "meta.Sources.File.DataSourceIP": "Data source IP",
+  "meta.Sources.File.ClusterName": "ClusterName",
```
Comment (healchow) [inline inlong-dashboard/src/locales/en.json:22]:
```suggestion
  "meta.Sources.File.ClusterName": "Cluster Name",
```

File: inlong-dashboard/src/metas/sources/defaults/File.ts [hunk 2]
```diff
@@ -27,13 +27,49 @@ import { SourceInfo } from '../common/SourceInfo';
 const { I18n } = DataWithBackend;
 const { FieldDecorator } = RenderRow;
 const { ColumnDecorator } = RenderList;
+let testId = 0;
```
Comment (healchow) [inline inlong-dashboard/src/metas/sources/defaults/File.ts:30]: testId?

**Review by leezng — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**